src/game/client/components/binds.cpp

```diff
@@ -326,15 +326,44 @@ void CBinds::ConBinds(IConsole::IResult *pResult, void *pUserData)
 {
 	CBinds *pBinds = (CBinds *)pUserData;
 	char aBuf[1024];
-	for(int i = 0; i < KEY_LAST; i++)
+	if(pResult->NumArguments() == 1)
 	{
-		for(int m = 0; m < MODIFIER_COUNT; m++)
+		char aBuf[256];
+		const char *pKeyName = pResult->GetString(0);
+
+		int Modifier;
+		int KeyID = pBinds->DecodeBindString(pKeyName, &Modifier);
+		if(!KeyID)
 		{
-			if(pBinds->m_aaaKeyBindings[i][m][0] == 0)
-				continue;
-			str_format(aBuf, sizeof(aBuf), "%s%s (%d) = %s", GetModifierName(m), pBinds->Input()->KeyName(i), i, pBinds->m_aaaKeyBindings[i][m]);
+			str_format(aBuf, sizeof(aBuf), "key '%s' not found", pKeyName);
 			pBinds->Console()->Print(IConsole::OUTPUT_LEVEL_STANDARD, "binds", aBuf);
 		}
+		else
+		{
+			if(pBinds->m_aaaKeyBindings[KeyID][Modifier][0] == 0)
+				str_format(aBuf, sizeof(aBuf), "%s (%d) is not bound", pKeyName, KeyID);
```

Comment:
Could you display the `modifier` here too like the other messages

Reply:
If I understand the code correctly ``pKeyName`` already includes the modifier.

Reply:
Then you should do it like that for the else case 2 lines below

Reply:
Good idea. That made the long line shorter :)

What about in the 0 argument case?